From 769e9868c17a471323b81b12cab851c9fd22baf4 Mon Sep 17 00:00:00 2001
From: Georgy Yakovlev <gyakovlev@gentoo.org>
Date: Mon, 15 Jun 2020 14:18:45 -0700
Subject: [PATCH] add missing termios.h include

on ppc, TCGETS relies on struct termios being complete, on other
architectures it does not.
so termios.h should be included before ioctl.h
---
 src/seccomp.c      |   1 +
 1 file changed, 1 insertion(+)

diff --git a/src/seccomp.c b/src/seccomp.c
index e667adf..296f5b3 100644
--- a/src/seccomp.c
+++ b/src/seccomp.c
@@ -33,6 +33,7 @@ FILE_RCSID("@(#)$File: seccomp.c,v 1.15 2020/05/30 23:56:26 christos Exp $")
 #if HAVE_LIBSECCOMP
 #include <seccomp.h> /* libseccomp */
 #include <sys/prctl.h> /* prctl */
+#include <termios.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 #include <fcntl.h>
-- 
2.27.0

